main.js

```diff
@@ -16,10 +16,10 @@ let perfStats = {
 
 Object.defineProperty(exports, "__esModule", { value: true });
 
-const extension = require('./out/src/extension');
+const extension = require('./dist/extension.bundle');
```

Comment:
I don't think you meant to check this in?

Reply:
Need to undo it here too. Usually I catch this... https://github.com/microsoft/vscode-azurefunctions/pull/3275/files#diff-58417e0f781b6656949d37258c8b9052ed266e2eb7a5163cad7b0863e6b2916a